### PR TITLE
refactor(dotenv): format test names

### PR DESCRIPTION
**Pull request by timreichen**
Ref: https://github.com/denoland/deno_std/issues/3754

### CODE REVIEW

File: dotenv/parse_test.ts [hunk 1]
```diff
@@ -142,7 +142,7 @@ Deno.test("parser", () => {
   );
 });
 
-Deno.test("with comments", () => {
+Deno.test("parse() checks comments", () => {
```
Comment (kt3k) [inline dotenv/parse_test.ts:145]:
```suggestion
Deno.test("parse() ignores comments", () => {
```

File: dotenv/load_test.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ Deno.test({
 });
 
 Deno.test({
-  name: "load when multiple files",
+  name: "load() loads multiple files",
```
Comment (kt3k) [inline dotenv/load_test.ts:32]:
```suggestion
  name: "load() works as expected when the multiple files are imported",
```

ref: https://github.com/denoland/deno_std/pull/1964

**Comment by kt3k** [comment]
many of `checks` usages sound strange to me.

**Comment by timreichen** [comment]
> many of `checks` usages sound strange to me.

I had another look and updated some test names.

**Review by iuioiua — APPROVED** [review]
LGTM!

**Pull request merged**